pointcloud/basic_octree.go

```diff
@@ -50,21 +51,19 @@ type basicOctreeNode struct {
 	maxVal   int
 }
 
-// NewBasicOctree creates a new basic octree with specified center, side and metadata.
-func NewBasicOctree(center r3.Vector, sideLength float64) (*BasicOctree, error) {
+// NewBasicOctree creates a new basic octree with specified center, side and confidenceThreshold.
+func NewBasicOctree(center r3.Vector, sideLength float64, confidenceThreshold int) (*BasicOctree, error) {
```

Comment:
bit late to the game and missing some context -- why is configuring a confidenceThreshold something we want to do?

Reply:
The purpose of this PR is to remove the magic numbers that were inside the collision function for pointclouds.  But we still want to be able to configure it.  Hence, the argument